crow/setup3.py

```diff
@@ -18,15 +18,15 @@
 except:
   eigen_flags = ""
 include_dirs=['include/distributions','include/utilities','contrib/include']
-if eigen_flags.startswith("-I"):
+if eigen_flags.startswith(b"-I"):
   include_dirs.append(eigen_flags[2:].rstrip())
 swig_opts=['-c++','-py3','-Iinclude/distributions','-Iinclude/utilities']
```

Comment:
Same here, I am wondering if these includes should use `os.path.join`, also this is a subset of the includes above, I am wondering if it is at all beneficial to abstract these into variables? This may be overkill, and it is probably better to be explicit about what includes are needed where rather than just throwing all of the includes everywhere.